### ISSUE TITLE
Screen reader is announcing the incorrect number of rows number as 8, even though there are only 7 rows available:A11y_Aspire Dashboard_Resources_Screenreader.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/aspire#10123]
### GitHubTags
#A11ySev2; #A11yTCS;#DesktopWeb; #Win32;#.NET Project system for CSharp and VB;#BM_AspireDashboard_Web_June25;#Win32;#WCAG1.3.1;#ScreenReader;#NVDA;#Narrator;#Element:Table;#Info and Relationships;#A11yMAS;

### Actual Result:
Screen reader is announcing the incorrect number of rows number as 8, even though there are only 7 rows available.

### Expected Result:
The screen reader should correctly announce the rows number as 7, for example: 2 of 7, 3 of 7.

### Environment Details:
Application Name: Aspire Dashboard
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.4351

### Repro Steps:

1. Open Visual studio.
2. Open a aspire project and run it.
3. A new window will open in the chromium edge, TAB to "Resources" tab item and hit ENTER key.
4. Screen reader is announcing the incorrect number of rows number as 8, even though there are only 7 rows available.

### User Impact:
Visually impaired users who rely on screen readers may experience confusion because the screen reader is incorrectly announcing the number of rows as 8. This discrepancy can lead to misunderstandings and navigation difficulties for these users.

### Attachment:

![Image](https://github.com/user-attachments/assets/5d7081df-695f-482a-8102-57023da024e6)

**Comment by adamint** [comment]
This is a bug in `FluentDataGrid`'s virtualization option, which renders up to 2 empty rows.
Edit: `FluentDataGrid` is following the `Virtualize.SpacerElement` guidance and using a `<tr>`. The issue is actually with the Virtualize component itself.

**Comment by adamint** [comment]
This bug should be closed as external to Blazor itself 
cc @danmoseley @AnKushSingh05

**Comment by AnKushSingh05** [comment]
> This bug should be closed as external to Blazor itself cc [@danmoseley](https://github.com/danmoseley) [@AnKushSingh05](https://github.com/AnKushSingh05)

@adamint if the bug is external to blazor Could you please provide the service tree and ADO details so we can route the bug to external team.

**Comment by adamint** [comment]
Blazor has fixed this for net10, going to mark as completed. Note that the fix should be present in the next preview version of net 10.
